### PR TITLE
#48794 Automatically link plugin action to the latest process version

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major10/10.2.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -14,11 +14,10 @@ The following features were added:
   This plugin facilitates authentication for the Notificaties API plugin when interfacing 
   with [OpenNotificaties](https://github.com/open-zaak/open-notificaties).
 
-* **New feature2**
-
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+* **Automatically migrates plugin actions**
```
Comment (theo-ritense) [inline release-notes/major10/10.2.0/valtimo-backend-libraries.md:17]:
```suggestion
* **Automatically migrates linked plugin actions**
```

File: release-notes/major10/10.2.0/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -14,11 +14,10 @@ The following features were added:
...
 
+  Plugin actions that are linked to the latest process version will now automatically be linked to a new version of
+  the process that is deployed.
```
Comment (theo-ritense) [inline release-notes/major10/10.2.0/valtimo-backend-libraries.md:20]:
```suggestion
  the process that is being deployed.
```

File: release-notes/major10/10.2.0/valtimo-backend-libraries.md [hunk 3]
```diff
@@ -14,11 +14,10 @@ The following features were added:
   This plugin facilitates authentication for the Notificaties API plugin when interfacing 
   with [OpenNotificaties](https://github.com/open-zaak/open-notificaties).
 
-* **New feature2**
-
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+* **Automatically migrates linked plugin actions**
```
Comment (marijnritense) [inline release-notes/major10/10.2.0/valtimo-backend-libraries.md:17]: migrates -> migrate

Also, is this an always-on feature, or is it possible to opt out of this functionality?

Comment (Klaas-Ritense) [reply release-notes/major10/10.2.0/valtimo-backend-libraries.md]: Changed. Also, this is always on.

**Review by marijnritense — APPROVED** [review]

**Pull request merged**